### DISCUSSION TITLE
Alimentation automatique des actifs

### DISCUSSION THREAD

**Discussion by DavidSecu** [dbarzin/mercator#1010] [General]
Bonjour,
Je n'ai pas encore testé ce produit, pourriez vous répondre à ma question ci-dessous ? (Je ne l'ai pas trouvé dans mes recherches)
Est-il possible d'intégrer les actifs (serveurs physiques ou virtuelles, logiciels, etc) de façon automatique ?

**Comment by dbarzin** [comment]
Il existe une [API](https://dbarzin.github.io/mercator/api/) avec des exemples d'utilisation dans différents langages.

Il est possible également d'importer des données avec la commande [load data](https://dev.mysql.com/doc/refman/8.0/en/load-data.html) ou avec [MySQL Workbench](https://dev.mysql.com/doc/workbench/en/wb-admin-export-import-table.html) ou encore [Heidi SQL](https://www.heidisql.com/help.php#import).
